src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx

```diff
@@ -2,6 +2,7 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import moment from 'moment';
 import classNames from 'classnames';
+import buildFormlett from '../ViewDependentsFormStates/BuildFormlett.jsx';
```

Comment:
I don't think we need the file extension here

Reply:
ugh, yeah Atom autofills these and I don't always catch them. Updated.